packages/chaire-lib-backend/src/utils/processManagers/OSRMProcessManager.ts

```diff
@@ -130,7 +130,7 @@ const start = function(parameters = {} as any): Promise<any> {
     return new Promise((resolve, reject) => {
         const command = 'osrm-routed';
         const commandArgs = [
-            osrmDirectoryPath + '/*.osrm',
+            osrmDirectoryPath + `/${mode}.osrm`,
```

Comment:
The other comment applies to this line, not the one from OSRMServicePreparation, sorry.